src/components/Download.tsx

```diff
@@ -1,10 +1,232 @@
+import { useState, useMemo } from 'react';
+import {
+  Button,
+  Alert,
+  Collapse,
+  Typography,
+  List,
+  ListItem,
+  Switch,
+  FormControlLabel,
+  Link,
+} from '@mui/material';
+import Ajv from 'ajv';
 import NavigationButton from './NavigationButton';
+import DataDictionaryPreview from './DataDictionaryPreview';
+import useDataStore from '../stores/data';
+import schema from '../assets/neurobagel_data_dictionary.schema.json';
+import { DataDictionary } from '../utils/types';
+import emoji from '../assets/download-emoji.png';
 
 function Download() {
+  const [dictionaryCollapsed, setDictionaryCollapsed] = useState(false);
+  const [forceAllowDownload, setForceAllowDownload] = useState(false);
+
+  const uploadedDataTableFileName = useDataStore((state) => state.uploadedDataTableFileName);
+
+  const columns = useDataStore((state) => state.columns);
+
+  const dataDictionary = useMemo(
+    () =>
+      Object.entries(columns).reduce((acc, [_, column]) => {
```

Comment:
```suggestion
      Object.entries(columns).reduce((acc, [_columnName, columnAnnotations]) => {
```
I need to guess / remember a good bit here otherwise